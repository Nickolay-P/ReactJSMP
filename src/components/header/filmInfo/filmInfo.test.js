

import React from 'react';
import renderer from 'react-test-renderer';
import {FilmInfo} from './filmInfo.js';


it('FilmInfo rendered correctly', () => {
  const AppComponent = renderer.create(<FilmInfo />).toJSON();
  expect(AppComponent).toMatchSnapshot();
});