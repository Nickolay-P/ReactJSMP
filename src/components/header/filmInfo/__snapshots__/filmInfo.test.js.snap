// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmInfo rendered correctly 1`] = `
<div>
  <div
    className="headerTop"
  >
    <div
      className="rouletteLabel"
    >
      netflixroulette
    </div>
    <button
      className="headerSearchButton"
    >
      SEARCH
    </button>
  </div>
  <div
    className="headerMiddle"
  >
    <div
      className="headerTitleImage"
    >
      <img
        alt="LOTR1"
        src="../../public/img/L1.jpg"
      />
    </div>
    <div
      className="headerTitleDetails"
    >
      <div
        className="headerTitleName"
      >
        The Lord of the Rings
      </div>
      <div
        className="headerTitleCategory"
      >
        No requirements so f... ...
      </div>
      <div
        className="headerTitleYearAndDuration"
      >
        <div
          className="headerTitleYear"
        >
          2001
        </div>
        <div
          className="headerTitleDuration"
        >
          228 min
        </div>
      </div>
      <div
        className="headerTitleDescription"
      >
        The Lord of the Rings: The Fellowship of the Ring is a New Zealand-American 2001 epic fantasy adventure film directed by Peter Jackson based on the first volume of J. R. R. Tolkien's The Lord of the Rings. It is the first instalment in The Lord of the Rings series, and was followed by The Two Towers (2002) and The Return of the King (2003), based on the second and third volumes of The Lord of the Rings.
      </div>
    </div>
  </div>
</div>
`;
